examples/example_atomic.c

```diff
@@ -6,10 +6,10 @@
 #include <linux/module.h>
 
 #define BYTE_TO_BINARY_PATTERN "%c%c%c%c%c%c%c%c"
-#define BYTE_TO_BINARY(byte)                                                   \
-    (byte & 0x80 ? '1' : '0'), (byte & 0x40 ? '1' : '0'),                      \
-        (byte & 0x20 ? '1' : '0'), (byte & 0x10 ? '1' : '0'),                  \
-        (byte & 0x08 ? '1' : '0'), (byte & 0x04 ? '1' : '0'),                  \
+#define BYTE_TO_BINARY(byte)                                  \
```

Comment:
Why did you change the indention?

Reply:
Because in my ubuntu, when I generated the pdf file, the line break macro will go to the next line.

![](https://imgur.com/oLSFhRG.png)

it can avoid by setting `AlignEscapedNewlines: Left`.

Reply:
Don't change the style in this pull request. You can submit another one to concentrate on further style tweaks.